composer.json

```diff
@@ -3,25 +3,24 @@
     "description": "Shell for PHPCR",
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/console": "~2.8|~3.0",
+        "symfony/console": "~2.8|~3.4|~4.0",
         "jackalope/jackalope": "~1.1",
         "phpcr/phpcr": "~2.1",
-        "phpcr/phpcr-utils": "~1.2",
-        "symfony/finder": "~2.8|~3.0",
-        "symfony/serializer": "~2.8|~3.0",
-        "symfony/yaml": "~2.8|~3.0",
-        "symfony/dependency-injection": "~2.8|~3.0",
-        "symfony/expression-language": "~2.8|~3.0",
+        "phpcr/phpcr-utils": "~1.2", 
+        "symfony/finder": "~2.8|~3.4|~4.0",
+        "symfony/serializer": "~2.8|~3.4|~4.0",
+        "symfony/yaml": "~2.8|~3.4|~4.0",
```

Comment:
why 3,4?

Reply:
indeed could be 3.0 i will change that here

Reply:
sorry, imho this should be 3.4 indeed. all other 3.x versions of symfony are end of life and don't even get security fixes anymore: https://symfony.com/roadmap